### ISSUE TITLE
Licenses not visible after SBOM upload

### ISSUE THREAD

**Issue by vasba** [DependencyTrack/dependency-track#2226]
### Current Behavior

If the attached SBOM is loaded into Dependency Track the licenses are not visible.

[bom.txt](https://github.com/DependencyTrack/dependency-track/files/10140256/bom.txt)

### Steps to Reproduce

1.Load the file
2. Check the project and no licenses are shown

### Expected Behavior

The licenses should be visible in the project for each component.

### Dependency-Track Version

4.6.2

### Dependency-Track Distribution

Container Image

### Database Server

PostgreSQL

### Browser

Mozilla Firefox

### Checklist

- [X] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [X] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this defect was already reported

**Comment by stevespringett** [comment]
> I'm not sure about Dependency Track handling license expressions.

Yes, ideally it should. One of the early tickets that nobody has had a chance to work on yet.#170

**Comment by syalioune** [comment]
OK, i'll try to work on it (deserializer part and DT handling of license expressions)

**Comment by vasba** [comment]
Hi!

Great that you found this issue. I will try again when this will be fixed.

Thanks!

**Comment by JoshuaAtEriks** [comment]
encountered the same issue while uploading a SBOM generated by Aqua Trivy. The license info in SBOM json is of below format 
`      "licenses": [
        {
          "expression": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0"
        }
      ]`

**Comment by strowi** [comment]
Looking forward to a fix, would love to use trivy for the sbom-generation.

**Comment by LesSyner** [comment]
I've verified things on trivy side and have found issues there regarding invalidity of generated SBOM files according to CycloneDX-JSON schema. Here is issue I've created for trivy: [https://github.com/aquasecurity/trivy/issues/4900](https://github.com/aquasecurity/trivy/issues/4900)
It looks like issue is in trivy. I've tried SBOM files generated by other tools like syft and they are valid according to schema and when imported to Dependency-track they show properly licenses.
